homeassistant/components/lyric/__init__.py

```diff
@@ -85,15 +85,15 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     client = ConfigEntryLyricClient(session, oauth_session)
 
     client_id = hass.data[DOMAIN][CONF_CLIENT_ID]
-    lyric: Lyric = Lyric(client, client_id)
+    lyric = Lyric(client, client_id)
 
     async def async_update_data() -> Lyric:
         """Fetch data from Lyric."""
         try:
             async with async_timeout.timeout(60):
                 await lyric.get_locations()
             return lyric
-        except (*LYRIC_EXCEPTIONS, TimeoutError) as exception:
+        except (*LYRIC_EXCEPTIONS, asyncio.TimeoutError) as exception:
```

Comment:
```suggestion
        except LYRIC_EXCEPTIONS as exception:
```

Already handled by the coordinator: https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/update_coordinator.py#L151